library/alloc/src/vec.rs

```diff
@@ -3026,7 +3026,10 @@ impl<T> Drain<'_, T> {
     }
 }
 
-/// An iterator produced by calling `drain_filter` on Vec.
+/// A draining iterator with filter predicate for `Vec`.
+///
+/// This struct is created by [`Vec::drain_filter`].
```

Comment:
```suggestion
/// This struct is created by [`Vec::drain_filter()`].
```

Reply:
I just followed the one in `Drain` which does not have brackets.

Reply:
Hmm, do you mind updating `Drain` to use parentheses as well? Or if this is common to not have `()` it's fine to leave to a follow-up PR.

Reply:
I personally think not having `()` is good enough. Last time we added this `()` because there is a word `function`, there isn't any here. Can you think of anything `Vec::drain_filter` be other than a function?

Reply:
Makes sense to me. The only other thing it could be is an associated constant or type, which is clear it's not by the naming and context.